scenario03/jmeter/01-Scenario-03-ProvisionFederatedUsersBytheIdentityProvider.jmx

```diff
@@ -669,7 +669,7 @@
                &lt;/xsd:properties&gt;&#xd;
                &lt;xsd:properties&gt;&#xd;
                   &lt;xsd:name&gt;callBackUrl&lt;/xsd:name&gt;&#xd;
-                  &lt;xsd:value&gt;https://${isCallbackHost}:${isPort}/commonauth&lt;/xsd:value&gt;&#xd;
+                  &lt;xsd:value&gt;https://${isCallbackHost}/commonauth&lt;/xsd:value&gt;&#xd;
```

Comment:
port missing ?

Reply:
Port needs to be removed as we are setting the host name as is.qa.com in the callback url

Reply:
fine, we can go ahead with this